guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -24,12 +24,13 @@ View compliance reports.
 | Edit partition tables | View, create, edit and destroy partition tables.
 | Manager | A role similar to administrator, but does not have permissions to edit global settings.
 In the {ProjectWebUI}, global settings can be found under *Administer* > *Settings*.
+This role is one of two with the ability to grant the `execute_jobs_on_infrastructure_hosts` permission.
 | Organization admin | An administrator role defined per organization.
 The role has no visibility into resources in other organizations.
 | Red{nbsp}Hat Access Logs | View the log viewer and the logs.
-| Remote Execution Manager | A role with full remote execution permissions, including modifying job templates.
-| Remote Execution User | Run remote execution jobs.
-| Site manager | A restrained version of the Manager role.
+| Remote Execution Manager | A role with full remote execution permissions, excluding {ProjectName} and {Capsules} hosts, and including modifying job templates.
+| Remote Execution User | Run remote execution jobs against hosts, excluding {ProjectName} and {Capsules}.
```

Comment:
same as above.

Reply:
The voice and mood changes in this sentence and breaks parallelism with the other examples.

Reply:
@melcorr , the info in this was suggested by the developers to highlight the new REX capabilities.  Are you recommending I take out the parts like "excluding XYZ hosts and including modifying..." to make it less specific?

Reply:
Information comes from developers. The writing style is the remit of the writer. 
What I am saying is to follow our style with regards to parallelism.